firmware_configurations/klipper/eddie/Voron2_SKR_Universal_Config.cfg

```diff
@@ -379,7 +378,6 @@ heater_temp: 45.0
 heater_pin: z:P2.3
 sensor_type: NTC 100K MGB18-104F39050L32
 sensor_pin: z:P0.23
-smooth_time: 3.0
```

Comment:
the default in klipper is 2 seconds here. So removing this only puts it back to 2 seconds.  From what I've seen on user systems on Discord, 0.5 seconds gives far better results.